latest/src/app/documentation/demos/full-page-layouts/basepage-layout/basepage-layout.demo.html

```diff
@@ -1,20 +1,23 @@
 <clr-main-container>
     <clr-demo-menu></clr-demo-menu>
-    <clr-history-pinned *ngIf="withHistory" [clrUsername]="'admin'" [clrContext]="context" [clrDomain]="'porscheinformatik.github.io'"></clr-history-pinned>
```

Comment:
the explicit parameter for domain was needed on the website, as it's not allowed to set top level cookies. is there a reason to remove that?

Reply:
Did this so that it works locally - didn't know that about GitHub pages